Clicker/ViewControllers/PollsViewController+Extension.swift

```diff
@@ -80,7 +80,7 @@ extension PollsViewController: PollsCellDelegate {
                             }
                         }
                     }
-                    let pollsDateViewController = PollsDateViewController(delegate: self, pollsDateArray: pollsDateArray, session: session, userRole: userRole)
+                    let pollsDateViewController = PollsDateViewController(delegate: self, pollsDateArray: pollsDateArray.reversed(), session: session, userRole: userRole)
```

Comment:
why are we reversing it?

Reply:
Mindy fixed this in her pr but the polls weren't displaying newest to oldest